test/functional/test_framework/mintlayer/utxo.py

```diff
@@ -82,6 +82,37 @@ def get_staking_ledger(self):
 
         return ((h, o.value) for (h, o) in query)
 
+    # TODO: move to a separate file
+    """ accesses current era """
+    def current_era(self):
```

Comment:
I think you can just simply return the result without storing it to a temporary variable